frontend/src/store/modules/shoots/index.js

```diff
@@ -366,14 +383,23 @@ const mutations = {
   HANDLE_EVENTS (state, { rootState, rootGetters, events }) {
     const onlyShootsWithIssues = get(state, 'shootListFilters.onlyShootsWithIssues', true)
     let setFilteredItemsRequired = false
+    const objectsToAdd = []
+    const shouldAddShoot = (shoot) => {
+      // Do not add healthy shoots when onlyShootsWithIssues=true, this can happen when toggeling flag
+      return rootState.namespace !== '_all' ||
+      !onlyShootsWithIssues ||
+      onlyShootsWithIssues === shootHasIssue(shoot)
+    }
```

Comment:
If the implementation would follow what is described in the comment the comment would be not necessary.
```suggestion
    const doNotAddShoot = shoot => rootState.namespace === '_all' && onlyShootsWithIssues && !shootHasIssue(shoot)
```